src/client/components/Graphs.tsx

```diff
@@ -14,6 +14,39 @@ const parseDate = (timestamp: string): string => {
   return `${y}-${m}-${d}T${h}:${min}`
 }
 
+export const dataParser = (rawData: any, provider: string): any => {
+  let parsedData: any[] = []
+  const shortDateData: any[] = []
```

Comment:
As much as possible, try to avoid using `any`. If you find yourself needing to use `any`, you may need to think more about what the data/data structure looks like. Are there types or interfaces that already exist in the packages or in other files? If so, you could always import them.